website/content/en/docs/setup/going-to-prod/arch/aggregator.md

```diff
@@ -93,7 +93,6 @@ Partitioning, or “topics” in Kafka terminology, refers to separating data in
 
 #### Recommendations
 
-- If you’re using your pub-sub system purely for durability, consider Vector’s external buffers, which buffers data on your pub-sub system while transparently handling data producing and consuming.
```

Comment:
I think in another place where we mention a not-yet-present feature, was a note that it was WIP. Do we want to do that again, rather than remove it all?

Reply:
🤔 we don't have any imminent plans for this so I'd rather just remove it and we can re-add if/when we pick this work up.